modules/ensemble/lib/widget/button.dart

```diff
@@ -115,6 +118,9 @@ class ButtonController extends BoxController {
 
   IconModel? startingIcon;
   IconModel? endingIcon;
+
+  bool? loading;
+  YamlMap? loadingWidget;
```

Comment:
use `dynamic` type. This may throw an exception if we put anything but Map into the definition.